api/pom.xml

```diff
@@ -8,7 +8,7 @@
         <relativePath>../../../microservice-parent/pom.xml</relativePath>
     </parent>
     <artifactId>dictionary-api</artifactId>
-    <version>4.0.5-SNAPSHOT</version>
+    <version>4.0.5</version>
```

Comment:
Has 4.0.5, been released? Typically we would change this to do the release then set it to the next snapshot.

Reply:
I will fix this up when merging it in.